app/models/manageiq/providers/vmware/infra_manager/provision/cloning.rb

```diff
@@ -58,6 +59,15 @@ def dest_resource_pool
     cluster ? cluster.default_resource_pool : dest_host.default_resource_pool
   end
 
+  def dest_storage_profile
+    # Storage Profiles were added in vSphere 5.5 so make sure we
+    # don't send newer api types to an older vCenter
+    return nil if source.ext_management_system.api_version < '5.5'
```

Comment:
We do the same check before attempting to login to the endpoint:
https://github.com/ManageIQ/manageiq/blob/master/gems/pending/VMwareWebService/MiqPbmInventory.rb#L7